Umbraco-Cloud/Frequently-Asked-Questions/index.md

```diff
@@ -135,11 +134,7 @@ Please contact us using the chat button at the bottom right corner of the [Umbra
 
 ### Does Umbraco Cloud support Let's Encrypt certificates?
 
-Yes, any certificate that can be converted into a `pfx` file is supported on  Cloud.
-
-However, we currently don't have an automated way to generate a Let's Encrypt certificate for you and bind it to your site. This means that you'll be responsible for generating a certificate every 90 days (the maximum validity period of a Let's Encrypt certificate) and updating your site with the new certificate.
-
-Since it is easy to forget this, we recommend you use certificate monitoring tools on your site to remind you to do this.
+Yes, we have now launched [Umbraco Latch](https://umbraco.com/blog/introducing-umbraco-latch/) which automates the process of installing and renewing Let's Encyrpt certificates, all new sites are automatically setup with a Let's Encyrpt certificate and HTTPS enabled by default. 
```

Comment:
This is great, Chris!
One thing: Wouldn't it make more sense to link to the documentation article about Latch: https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Manage-Domains/Umbraco-Latch ?
This article also references the blog post you've linked to.